tests/run_tests.sh

```diff
@@ -36,17 +36,21 @@ flake8 --select BLK conflicting_configurations/*.py
 flake8 --select BLK with_bad_toml/hello_world.py --black-config ''
 
 echo "Checking we report expected black changes"
-diff test_changes/hello_world.txt <(flake8 --select BLK test_changes/hello_world.py)
-diff test_changes/hello_world_EOF.txt <(flake8 --select BLK test_changes/hello_world_EOF.py)
-diff test_changes/hello_world_EOF.txt <(flake8 --select BLK test_changes/hello_world_EOF.py --black-config '')
-diff with_bad_toml/hello_world.txt <(flake8 --select BLK with_bad_toml/hello_world.py)
-diff with_pyproject_toml/ignoring_toml.txt <(flake8 with_pyproject_toml/ --select BLK --black-config '')
+diff --strip-trailing-cr test_changes/hello_world.txt <(flake8 --select BLK test_changes/hello_world.py)
```

Comment:
Can we not tweak the Windows git ``core.autocrlf`` newline settings instead?

Reply:
I was thinking if git is told to do the line ending conversion automatically, then plain ``diff`` should just work even on Windows?